### PR TITLE
TNO-2336 Send email on failure

### PR DESCRIPTION
**Pull request by fbarreta**
Send email on service failure.

I came with a no email format. If it's OK. I'll replicate into the other services.

### CODE REVIEW

File: services/net/folder-collection/FolderCollectionManager.cs [hunk 1]
```diff
@@ -64,20 +78,63 @@ public class FolderCollectionManager : ServiceManager<FolderCollectionOptions>
...
+    public async Task SendEmailAsync(string subject, string message)
+    {
+        if (this.Options.SendEmailOnFailure)
+        {
+            try
+            {
+                var email = new TNO.Ches.Models.EmailModel(this.ChesOptions.From, this.Options.EmailTo, subject, message);
+                await this.Ches.SendEmailAsync(email);
+            }
+            catch (ChesException ex)
```
Comment (Fosol) [inline services/net/folder-collection/FolderCollectionManager.cs:114]: Does the ChesService ever throw any other type of error?  If it does you'll need to handle it, or this process can break other processes.

**Comment by Fosol** [comment]
Do we need this anymore?

File: services/net/folder-collection/FolderCollectionManager.cs (hunk 1 above)
Comment (fbarreta) [reply services/net/folder-collection/FolderCollectionManager.cs:114]: No, we can remove this PR.

**Comment by fbarreta** [comment]
This changes will be addressed on this pr:
https://github.com/bcgov/tno/pull/1579

**Pull request closed without merging**